engine/table/src/main/java/io/deephaven/engine/util/LayoutHintBuilder.java

```diff
@@ -480,12 +592,19 @@ public String build() {
             sb.append("groupable=").append(String.join(",", groupableColumns)).append(';');
         }
 
+        if (columnGroups != null && !columnGroups.isEmpty()) {
+            sb.append("columnGroups=");
+            List<String> groupStrings =
+                    columnGroups.values().stream().map(ColumnGroup::serialize).collect(Collectors.toList());
```

Comment:
better: skip the list, and just use Collectors.joining(...) to emit a single string here.

or, iterate as you walk the stream and append to the stringbuilder:

```
...map(ColumnGroup::serialize).forEach(str -> sb.append("|").append(str));
```